hw3/templates/index.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1 class="title"> Регистрация </h1>
+<div class="block">
+    <form method="post" action="{{ url_for('index') }}">
+        {{ form.csrf_token }}
+        <p style="text2">
+            {{ form.name.label }}<br>
+            {{ form.name(size=32) }}
+        </p>
+        <p style="text2">
+            {{ form.surname.label }}<br>
+            {{ form.surname(size=32) }}
+        </p>
+        <p style="text2">
+            {{ form.email.label }}<br>
+            {{ form.email(size=32) }}
+        </p>
+        <p style="text2">
+            {{ form.password.label }}<br>
+            {{ form.password(size=32) }}
+        </p>
+        <p style="text2">
+            {{ form.confirm_password.label }}<br>
+            {{ form.confirm_password(size=32) }}
+        </p>
+        <p style="text2">
+            <input type="submit" value="Register">
+        </p>
```

Comment:
Очень много повторений, не соблюден принцип DRY
```suggestion
<p>
    {% for field in form if field.name != 'csrf_token' %}
    <p>
        {{ field.label }}<br>
        {{ field(size=32) }}
    </p>
    <p>
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
    </p>
    {% endfor %}
</p>
```
Причем строки
```html
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
```
будут повторять flash сообщение об ошибке. Вы можете решить, что лучше: либо выводить сообщение тут, либо выше.